config/locales/ca.yml

```diff
@@ -25,5 +25,20 @@ ca:
             client_secret: Secret de client
             site: URL del servidor d'Omniauth
           enabled_by_default: Activat per defecte
+    verifications:
+      civicrm:
+        errors:
+          invalid_method: No us heu pogut verificar. El mètode de verificació OAuth2 no era "civicrm" tal com s'esperava.
+          invalid_status: No us heu pogut verificar. La verificació OAuth2 ha estat correcta.
+        invalid:
+          email_intro: No s'ha pogut concedir l'autorització de CiViCRM.
+          email_outro: Si us plau, poseu-vos en contacte amb el suport de la vostra plataforma per comprovar l'error que s'ha produït.
```

Comment:
```suggestion
          email_outro: Si us plau, poseu-vos en contacte amb el suport de la vostra plataforma per comprovar l'error que ha ocorregut.
```

Reply:
I think I would rather leave this one as it is.